capitalizedecorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
```

Comment:
- [ ] Here you have to `Capitalize` the name and not make it to the uppercase whole word. There is a `capitalize` method you can use instead of `upcase`.